src/basic/copy.c

```diff
@@ -1103,7 +1103,7 @@ int copy_file_atomic_full(
         assert(from);
         assert(to);
 
-        /* We try to use O_TMPFILE here to create the file if we can. Note that that only works if COPY_REPLACE is not
+        /* We try to use O_TMPFILE here to create the file if we can. Note that it only works if COPY_REPLACE is not
```

Comment:
"It" refers to "file", so this sentence doesn't make sense now. "Note that this only work ..." maybe?